app/src/test/kotlin/no/nav/tiltakspenger/felles/NavkontorTest.kt

```diff
@@ -4,56 +4,57 @@ import io.kotest.assertions.throwables.shouldThrow
 import org.junit.jupiter.api.Test
 
 class NavkontorTest {
```

Comment:
Jeg vurderte å fjerne denne testen, og kanskje også valideringen siden dette ikke lenger er input fra saksbehandler. Skal jeg fjerne den?

Reply:
Ja, jeg er enig i at vi ikke trenger ha veldig sterk validering når det bare kommer fra et annet register. Vi kan legge på validering og test hvis vi lar saksbehandler overstyre dette senere. Bør vi ha en begrensing på lengde så vi ikke for oom? Eller stoler vi blindt på dataene som kommer.

Reply:
Jeg tenker vi kan stole på dataene som kommer i dette tilfellet.